src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const items = sourceString.split(';');
+  const result = {};
+
+  for (const item of items) {
+    const params = item.split(':');
+
+    if (params.length === 2) {
```

Comment:
The current implementation assumes that each item will have exactly one colon. Consider adding error handling for cases where `params.length` is not equal to 2, to handle malformed input gracefully.